### DISCUSSION TITLE
common code style for all Android modules

### DISCUSSION THREAD

**Discussion by Malachiasz** [LemonAppDev/konsist#1557] [Ask a Question]
### Wanted end result.

I would like to write common rules for all modules. How to achieve?

### What you tried so far.

I managed to only write rule for a single module.

**Comment by igorwojda** [comment]
Create a proper scope that includes all files from project - `Konsist.scopeFromProduction()` (all non-test files) or `Konsist.scopeFromProject()` (all Kotlin files in project)

```kotlin
Konsist
   .scopeFromProject() // includes all files in scope
   .classes()  // rules are applied to classes present in above scope
   .assertTrue {
       // ....
   }
```

https://docs.konsist.lemonappdev.com/writing-tests/koscope
